x/marker/client/cli/tx.go

```diff
@@ -177,14 +177,14 @@ with the given supply amount and denomination provided in the coin argument
 // GetCmdMint implements the mint additional supply for marker command.
 func GetCmdMint() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:     "mint [coin]",
+		Use:     "mint [coin] [recipient]",
 		Aliases: []string{"m"},
-		Args:    cobra.ExactArgs(1),
+		Args:    cobra.ExactArgs(2),
```

Comment:
The recipient needs to be optional.

```suggestion
		Args:    cobra.RangeArgs(1, 2),
```

Then, below, only parse and set the recipient if `len(args) > 1`.